common-pipeline/datarecord-metadata/datarecord-metadata-v3-schema.json

```diff
@@ -149,7 +149,7 @@
           "minItems": 0,
           "items": {
             "description": "A value discovered in the data",
-            "type": "string"
+            "type": "Can be a primitive type (string, integer, double, date, time, timestamp, binary, boolean)"
```

Comment:
This will need to be an array https://json-schema.org/understanding-json-schema/reference/type.html.  `type` is used for schema validation so we can't just put a string description here.